src/components/hand-controls.js

```diff
@@ -175,13 +175,13 @@ module.exports.Component = registerComponent('hand-controls', {
     var controlConfiguration;
     var el = this.el;
     var hand = this.data;
+    var handModelOrientation = hand === 'left' ? Math.PI / 2 : -Math.PI / 2;
```

Comment:
We can move this declaration inside the model load handler:

https://github.com/aframevr/aframe/pull/4396/files#diff-02099d98da3e9b5683f5a6ac5092aec4R190

`